src/rprblender/export/light.py

```diff
@@ -45,7 +44,7 @@ def get_radiant_power(light: bpy.types.Light, area=0.0):
     if light.type in ('POINT', 'SPOT'):
         units = rpr.intensity_units_point
         if units == 'DEFAULT':
-            return color * math.sqrt(light.energy)  # to match cycles
+            return default_intensity * 0.01     # to match cycles
```

Comment:
Do we have any justification for this number?

Reply:
Experimented with different coefficients on different scenes. Tried to find more suitable.